corva/scheduled.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Callable
```

Comment:
name of this module is not descriptive. What "scheduled"? `runners/scheduled.py` will be more descriptive, since without visiting the code of the file itself, you can tell that it is implementation of scheduled runner

Reply:
thanks, moved both files to corva/runners/